openmc/statepoint.py

```diff
@@ -153,6 +153,9 @@ def __exit__(self, *exc):
         if self._summary is not None:
             self._summary._f.close()
 
+    def close(self):
+        self.__exit__()
```

Comment:
Probably clearer to do the other way around: put the current block of code from `__exit__` here, and then have `__exit__` be:
```Python
def __exit__(self, *exc):
    self.close()
```
Also, this method needs a docstring.

Reply:
I removed the method since I'm not using it anymore